### ISSUE TITLE
Running  ccdproc.subtract_overscan twice causes problems with FITS header cards.

### ISSUE THREAD

**Issue by stpstp** [astropy/ccdproc#427]
Running  ccdproc.subtract_overscan twice causes problems with FITS header cards.  I want to run  ccdproc.subtract_overscan first on the serial overscan for normal overscan subtraction, and then on the parallel overscan for smear correction (shutterless readout).

The data part looks fine after running subtract_overscan twice, but the fits header produces errors when I want to write a new fits file. Inspection of the header before writing to fits shows only the second suboscan info, and writeto results in:
raise VerifyError('CONTINUE cards must have string values.')
astropy.io.fits.verify.VerifyError: CONTINUE cards must have string values. This error only occurs when I run subtract_overscan twice, once and then trimming works OK.

Printing the header cards before writing to file, shows info for the second subtract_overscan, not for the first
HIERARCH subtract_overscan = 'suboscan' / Shortened name for ccdproc command    SUBOSCAN= 'ccd=<CCDData>, fits_section=[:,3077:3100], median=True, &'           CONTINUE  'overscan_axis=0'                                                     HIERARCH trim_image = 'trimim  ' / Shortened name for ccdproc command           TRIMIM  = 'ccd=<CCDData>, fits_section=[55:2100,3:3072]'               

```
from astropy.io import fits
from astropy import units as u
from ccdproc import CCDData

def CAL(ccd, biasSection='[2103:2148,:]',trimSection='[55:2100,3:3072]',smearSection='[:,3077:3100]'):
    ccd=ccdproc.subtract_overscan(ccd,median=True,overscan_axis=1, fits_section=biasSection)
    ccd=ccdproc.subtract_overscan(ccd,median=True,overscan_axis=0, fits_section=smearSection)
    ccd=ccdproc.trim_image(ccd, fits_section=trimSection)
    return(ccd)

hdu=fits.open(infile)
ccd=CCDData(hdu[channel].data,header=hdu[0].header+hdu[channel].header,unit=u.adu)
ccd1=CAL(ccd)
ccd1.write(outfile, clobber=True)
```
Version info:
Python 2.7.10 
astropy 1.2.1
ccdproc 1.1.0
numpy 1.11.2

Output
   ccd1.write(outfile, clobber=True)
  File "/usr/local/lib/python2.7/dist-packages/astropy/nddata/mixins/ndio.py", line 39, in write
    io_registry.write(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/registry.py", line 403, in write
    writer(data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/ccdproc/ccddata.py", line 705, in fits_ccddata_writer
    hdu.writeto(filename, **kwd)
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/hdu/hdulist.py", line 692, in writeto
    hdu._writeto(hdulist._file)
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/hdu/base.py", line 746, in _writeto
    header_offset, _ = self._writeheader(fileobj)
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/hdu/base.py", line 658, in _writeheader
    self._header.tofile(fileobj)
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/header.py", line 682, in tofile
    blocks = self.tostring(sep=sep, endcard=endcard, padding=padding)
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/header.py", line 630, in tostring
    s = str(card)
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/card.py", line 461, in __str__
    return self.image
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/card.py", line 767, in image
    self._image = self._format_image()
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/card.py", line 1212, in _format_image
    comment = self._format_comment()
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/card.py", line 1199, in _format_comment
    if not self.comment:
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/card.py", line 681, in comment
    self._comment = self._parse_comment()
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/card.py", line 1063, in _parse_comment
    for card in self._itersubcards():
  File "/usr/local/lib/python2.7/dist-packages/astropy/io/fits/card.py", line 1426, in _itersubcards
    raise VerifyError('CONTINUE cards must have string values.')
astropy.io.fits.verify.VerifyError: CONTINUE cards must have string values.

**Comment by crawfordsm** [comment]
We have updated how the header is updated in the development version, and I think this could be fixed in the most recent version.  I've tried it out and it seems to work, but could you check the development version against your data?

**Comment by stpstp** [comment]
Yes, the file writing is fixed when using version 1.3.dev17317  (and ccdproc 1.1.0
numpy 1.11.2 ). Thanks!
